commerce-test/src/testIntegration/java/com/liferay/commerce/shipment/test/CommerceShipmentTest.java

```diff
@@ -0,0 +1,1024 @@
+
+	@Before
+	public void setUp() throws Exception {
+		_company = CompanyTestUtil.addCompany();
+
+		_user = UserTestUtil.addUser(_company);
+
+		_commerceCurrency = CommerceCurrencyTestUtil.addCommerceCurrency(
+			_company.getGroupId());
+
+		_commerceOrders = new ArrayList<>();
+	}
+
+	@After
+	public void tearDown() throws Exception {
+		for (CommerceOrder commerceOrder : _commerceOrders) {
+			_commerceOrderLocalService.deleteCommerceOrder(commerceOrder);
+		}
+
+		_companyLocalService.deleteCompany(_company);
+	}
+
+	@Test(expected = CommerceOrderShippingAddressException.class)
+	public void testCheckoutWihoutShippingAddress() throws Exception {
+		frutillaRule.scenario(
+			"It should not be possible to create an order without shipping " +
+				"address"
+		).given(
+			"An order is created without a shipping address"
+		).when(
+			"Checkout is executed"
+		).then(
+			"An exception is raised"
+		);
+
+		CommerceChannel commerceChannel = _createCommerceChannel();
+
+		BigDecimal value = BigDecimal.valueOf(RandomTestUtil.nextDouble());
+
+		CommerceOrder commerceOrder = CommerceTestUtil.addB2CCommerceOrder(
+			_company.getGroupId(), _user.getUserId(),
+			_commerceCurrency.getCommerceCurrencyId(),
+			commerceChannel.getSiteGroupId());
+
+		CommerceShippingMethod commerceShippingMethod =
+			CommerceTestUtil.addFixedRateCommerceShippingMethod(
+				commerceOrder.getGroupId(), value);
+
+		commerceOrder.setCommerceShippingMethodId(
+			commerceShippingMethod.getCommerceShippingMethodId());
+
+		_commerceOrderLocalService.updateCommerceOrder(commerceOrder);
+
+		_commerceOrders.add(commerceOrder);
+
+		CommerceContext commerceContext = new TestCommerceContext(
+			commerceOrder.getCommerceCurrency(), commerceChannel, null, null,
+			null, commerceOrder);
+
+		_commerceOrderLocalService.checkoutCommerceOrder(
+			commerceOrder.getCommerceOrderId(), commerceContext,
+			ServiceContextTestUtil.getServiceContext(
+				commerceOrder.getGroupId()));
+	}
+
+	@Test
+	public void testCreateMultipleShippingForSameOrder() throws Exception {
+		frutillaRule.scenario(
+			"An order with multiple shippings is created"
+		).given(
+			"An order is created with a shipping associated"
+		).when(
+			"I try to add a second shipping to the same order"
+		).then(
+			"Multiple shipments are created"
+		);
+
+		CommerceChannel commerceChannel = _createCommerceChannel();
+
+		BigDecimal value = BigDecimal.valueOf(RandomTestUtil.nextDouble());
+
+		CommerceOrder commerceOrder =
+			CommerceTestUtil.createCommerceOrderForShipping(
+				_company.getGroupId(), _user.getUserId(),
+				_commerceCurrency.getCommerceCurrencyId(),
+				commerceChannel.getSiteGroupId(), value);
+
+		_commerceOrders.add(commerceOrder);
+
+		CommerceShipment commerceShipment =
+			CommerceShipmentTestUtil.createEmptyOrderShipment(
+				commerceOrder.getGroupId(), commerceOrder.getCommerceOrderId());
+
+		CommerceShipment commerceShipment1 =
```

Comment:
should be named `CommerceShipment commerceShipment1` and `CommerceShipment commerceShipment2`, rather than `CommerceShipment commerceShipment` and `CommerceShipment commerceShipment1`. See https://docs.google.com/document/d/188YFP0V4n9WJIPYZL3jGBRBvEsvBNmHOU97Fdd_g6Ew/edit#heading=h.urpkyx3jrv4y.